### ISSUE TITLE
Yandex 2

### ISSUE THREAD

**Issue by janosimas** [aunetx/text-translator#3]
Yandex has deprecated the V1 API and disabled the free tier keys.

**Comment by janosimas** [comment]
No, now Yandex requires a paid key. They removed the old api and the free tier.

**Comment by janosimas** [comment]
https://translate.yandex.com/developers/keys

**Comment by aunetx** [comment]
Oh OK so I guess I will need to pay in order to just test it... I guess the implementation will be left as it is for the moment, until I have the money (and motivation) to pay for it

**Comment by janosimas** [comment]
I think that if requested, they would give a free key for some time but they don't have a free tier anymore.
In any case, you should think if it's worth the effort.

for now, we should just [deprecate](https://doc.rust-lang.org/reference/attributes/diagnostics.html#the-deprecated-attribute) the Yandex api.

**Comment by aunetx** [comment]
Yeah that is what I will do, I guess that they should be OK...

And yes of course I will deprecate it tomorrow if it is possible for me :)
